code/datums/components/crafting/recipes.dm

```diff
@@ -142,6 +142,28 @@
 	category = CAT_WEAPONRY
 	subcategory = CAT_WEAPON
 
+/datum/crafting_recipe/reciever
+	name = "Modular Rifle Reciever"
+	tools = list(TOOL_WRENCH, TOOL_WELDER, TOOL_SAW)
+	result = /obj/item/weaponcrafting/receiver
+	reqs = list(/obj/item/stack/sheet/metal = 5,
+				/obj/item/stack/sticky_tape = 1,
+				/obj/item/screwdriver = 1,
```

Comment:
is this screwdriver supposed to be consumed, or just used as a tool?

Reply:
Consumed. The tip is essentially the actual hammer.